etc/tendrl/tendrl.conf.sample

```diff
@@ -16,3 +16,6 @@ log_level = DEBUG
 log_level = DEBUG
 tendrl_exe_file_prefix = /tmp/.tendrl_runner
 log_cfg_path = /etc/tendrl/node_agent_logging.yaml
+
+# deployment type can be either "production" or "dev"
+deployment_type = production
```

Comment:
This is wrong, a deployer should never have to define the type of environment to select the type of packaging that tendrl will enable